tests/test_remote_sse_resume.py

```
@@ -0,0 +1,277 @@
+"""SSE ``Last-Event-ID`` resume cursor.
+
+Plan §关键风险2: "SSE 断链自动重订阅；``/jobs/:id/events?resume=1`` 从最后
```

Ruff (0.15.10)

[warning] Docstring contains ambiguous `；` (FULLWIDTH SEMICOLON). Did you mean `;` (SEMICOLON)?

(RUF002)